### ISSUE TITLE
Is Indiff QtTabBar being maintained? 

### ISSUE THREAD

**Issue by Bamajon1974** [indiff/qttabbar#386]
Is the Indiff version of QtTabBar still being maintained? It is in beta stage and has not been updated in over a year. How does it differ from Quizo's version?

**Comment by qwop** [comment]
Yes, it is still under maintenance. You can refer to the documentation here. I will explain any updates here
https://www.yuque.com/indiff/qttabbar/rhyprn

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Bamajon1974 ***@***.***>
发送时间: Sunday, June 30, 2024 3:52:32 AM
收件人: indiff/qttabbar ***@***.***>
抄送: Subscribed ***@***.***>
主题: [indiff/qttabbar] Is Indiff QtTabBar being maintained? (Issue #386)


Is the Indiff version of QtTabBar still being maintained? It is in beta stage and has not been updated in over a year. How does it differ from Quizo's version?

―

**Comment by Bamajon1974** [comment]
Thank you for the quick response. I do have a couple of follow up questions...
1. Parts of the GUI are not in English. Is there a way to have the entire GUI, including plugins, fully translated? 
2. What is the difference between your QtTabBar and Quizo's QtTabBar? 
3. Your version of QtTabBar seems to have been in beta status for a while. What is the latest stable version? 
Thanks!

**Comment by Dhyfer1** [comment]
> 2. What is the difference between your QtTabBar and Quizo's QtTabBar? 

That's something I also want to know. 🤔

**Comment by Bamajon1974** [comment]
@qwop 

Thank you for the quick response. I do have a couple of follow up questions...

    Parts of the GUI are not in English. Is there a way to have the entire GUI, including plugins, fully translated?
    What is the difference between your QtTabBar and Quizo's QtTabBar?
    Your version of QtTabBar seems to have been in beta status for a while. What is the latest stable version?
    Thanks!

**Comment by Bamajon1974** [comment]
Thank you for the quick response. I do have a couple of follow up questions...

Parts of the GUI are not in English. Is there a way to have the entire GUI, including plugins, fully translated?
What is the difference between your QtTabBar and Quizo's QtTabBar?
Your version of QtTabBar seems to have been in beta status for a while. What is the latest stable version?
Thanks!

**Comment by Beet-Meat-Burger** [comment]
@Bamajon1974 @Dhyfer1

**Comment by Beet-Meat-Burger** [comment]
@Bamajon1974 @Dhyfer1 

The Chinese version of the repo has this section:

Why make this fork?

感谢大家讨论，确实是因为之前发布出去没有添加特别说明。 最开始维护的时候也只是自己用，后来开源之后有朋友也会用。 这边澄清一下qttabbar的一些历史：

软件最开始quizo维护，发布的；中间作者可能其他原因有一段时间没维护（据sf作者提到可能出车祸，不知真假！）
sf作者接手开始维护，因为quizo版本代码没有开源，所以sf作者是基于反编译的版本维护的。
但是2012~2013年的时候quizo回归开始维护，于是跟作者沟通过能不能一起合作，但是quizo拒绝了。导致sf作者最后一个beta版本没发布，就退出没有维护了。（那年sf作者还是大学生）
目前我维护的是基于这个没发布的版本代码，进行做过一些个性优化的版本。 因为本身专业是Java而不是C#，所以对非常专业的功能维护起来很吃力，比如 视频预览、win10黑暗主题、高DPI显示出问题等等
最后感谢大家对这款软件的讨论，我自己也是这个软件的深度用户。希望能力范围可以优化的更好。

Translated:
Thank you for discussion, it is indeed because it was released before it was released without adding a special explanation. When the first maintenance was maintained, it was only used by myself. Later, friends would use it after the open source. Clarify some of the history of qttabbar here:

The software was initially maintained and released by Quizo; the middle author may not be maintained for some other reasons for some time (according to the SF author mentioned that he might have had a car accident, I do not know if it is true!)
The SF author takes over and starts maintenance. Because the Quizo version code is not open source, the SF author is based on a decompiled version.
But from 2012 to 2013, Quizo returned to start maintenance, so he cooperated with the author to work together, but Quizo refused. As a result, the last Beta version of the SF author was not released, so he exited without maintenance. (The SF author was still college students at that year)
At present, I have maintained some personalized versions based on this unpublished version code. Because it is a java instead of C#, it is very difficult to maintain the very professional functions, such as video preview, Win10 dark theme, high DPI display problems, etc.
Finally, thank you for your discussion on this software. I am also a deep user of this software myself. The scope of hope can be optimized better.

So basically, someone decompiled QTTabBar, maintained it (also personalized it, added features), because they thought Quizo stopped development. This is a beta version of the decompiled version, and indiff is maintaining it.

**Comment by Bamajon1974** [comment]
Thank you for the reply. How does the stability of the indiff version of Qttabbar compare to the Quizo version?
